source/includes/harvest/_users.md

```diff
@@ -126,3 +126,44 @@ employee_id | The Employee ID of the user to retrieve
 
 <br>
 [See noteworthy response attributes.] (#the-user-object)
+
+## POST: Create User
```

Comment:
Hmm I think this should probably say Add User and not Create User to stay consistent with some of the creation endpoints for candidates. @tdphillipsjr, what language are you using for creating offices and departments?